Restore Sass variables for applied colours (#6437)
alphagov/govuk-frontend: CHANGELOG.md

| `$govuk-print-text-colour`          | `govuk-functional-colour(print-text)`          |
| `$govuk-secondary-text-colour`      | `govuk-functional-colour(secondary-text)`      |
| `$govuk-focus-colour`               | `govuk-functional-colour(focus)`               |
| `$govuk-focus-text-colour`          | `govuk-functional-colour(focus-text)`          |
| `$govuk-error-colour`               | `govuk-functional-colour(error)`               |
| `$govuk-success-colour`             | `govuk-functional-colour(success)`             |
| `$govuk-border-colour`              | `govuk-functional-colour(border)`              |
| `$govuk-input-border-colour`        | `govuk-functional-colour(input-border)`        |
| `$govuk-hover-colour`               | `govuk-functional-colour(hover)`               |
| `$govuk-link-colour`                | `govuk-functional-colour(link)`                |
| `$govuk-link-visited-colour`        | `govuk-functional-colour(link-visited)`        |
| `$govuk-link-hover-colour`          | `govuk-functional-colour(link-hover)`          |
| `$govuk-link-active-colour`         | `govuk-functional-colour(link-active)`         |

> Note that the Sass variables, now deprecated, are still available to facilitate your migration.

Reviewer: ```suggestion
Although we've now deprecated the Sass variables, they're still available to make your migration easier.
```
